src/styles/base.scss

```diff
@@ -19,6 +19,12 @@ body {
   width: 100%;
   height: 100%;
   overflow: hidden;
+
+  background: radial-gradient(
```

Comment:
Tested this on the exhibit computer, and it helped smooth the transition between screens a lot. 👍